src/hooks/v2/transactor/IssueTokensTx.ts

```diff
@@ -1,27 +1,42 @@
 import { V2ProjectContext } from 'contexts/v2/projectContext'
 import { V2UserContext } from 'contexts/v2/userContext'
 import { useContext } from 'react'
+import invariant from 'tiny-invariant'
 
-import { TransactorInstance } from '../../Transactor'
+import { onCatch, TransactorInstance } from '../../Transactor'
 
 export function useIssueTokensTx(): TransactorInstance<{
   name: string
   symbol: string
 }> {
-  const { transactor, contracts } = useContext(V2UserContext)
+  const { transactor, contracts, version } = useContext(V2UserContext)
   const { projectId } = useContext(V2ProjectContext)
 
   return ({ name, symbol }, txOpts) => {
-    if (!transactor || !projectId || !contracts?.JBTokenStore) {
-      txOpts?.onDone?.()
-      return Promise.resolve(false)
-    }
+    try {
+      invariant(
+        transactor && projectId && contracts?.JBController && name && symbol,
```

Comment:
Added name and symbol check here too. I know there's form validation, but just incase